README.md

```diff
@@ -177,6 +182,12 @@ steps:
 
 Since GitHub Enterprise Server uses self-hosted runners, dependencies are typically cached on the runner by whatever dependency management tool is being used (npm, maven, etc.).  This eliminates the need for explicit caching in some scenarios.
 
+## Changelog schedule and history 
```

Comment:
Should we make a separate file called CHANGELOG.md . Because this will make README unnecessarily long in future.

Reply:
I was thinking of the same. But, that would mean that the marketplace readme would not contain the changelog. So, I went this section to the bottom so that the current sections don't lose any visibility.

Reply:
ok looks good for now then.